modules/local/picrust.nf

```diff
@@ -0,0 +1,64 @@
+// Import generic module functions
```

Comment:
Wouldn't this make sense a proper nf-core/module? Or is that planned for a later release?

Reply:
Yes, that is the best candidate for a nf-core module in this pipeline. However, to my knowledge, it is only used with bacterial 16S amplicon sequencing (well, theoretically any marker gene), therefore pretty specific to this pipeline. But yes, if Picrust2 is used by any other pipeline, I'd make this into a proper nf-core module.